src/v2/view-builder/views/device-assurance-grace-period/DeviceAssuranceGracePeriodView.js

```diff
@@ -0,0 +1,41 @@
+import { loc, createCallout } from '@okta/courage';
+import { BaseForm, BaseView } from '../../internals';
+import EndUserRemediationMessages from '../end-user-remediation/EndUserRemediationMessages';
+import { getMessage } from 'v2/ion/i18nUtils';
+
+const ERROR_MESSAGE_CLASS = 'ERROR';
+const WARNING_MESSAGE_CLASS = 'WARNING';
+
+const DeviceAssuranceGracePeriodView = BaseForm.extend(
+  {
+    title() {
+      return loc('idx.device_assurance.grace_period.title', 'login');
+    },
+
+    save() {
+      return loc('idx.device_assurance.grace_period.continue_to_app', 'login');
+    },
+
+    showMessages() {
+      const messages = this.options.appState.get('messages').value;
+      const remediationMessages = messages.filter(message => message?.class === ERROR_MESSAGE_CLASS);
+      const warningMessage = messages.find(message => message?.class === WARNING_MESSAGE_CLASS);
```

Comment:
Are these two guaranteed to exist? i.e. will `remedationMessages.length > 0` and `warningMessage != null`?

Reply:
With the way I've written the code right now, I believe they will be guaranteed to exist but I'll add some null checking to be safe 👍